server/@types/ciagInductionApi/index.d.ts

```diff
@@ -530,4 +650,41 @@ export interface operations {
       }
     }
   }
+  /**
```

Comment:
This is the new request and response types that we are interested in for the new API endpoint